archetypes/Pools/FilterSelects/Bar.tsx

```diff
@@ -20,6 +21,13 @@ const REBALANCE_OPTIONS = [
         key: RebalanceEnum.last,
         text: 'Last Rebalance',
     },
+    {
+        key: RebalanceEnum.historic,
+        text: 'Historic Data',
+        disabled: {
+            optionKey: TooltipKeys.ComingSoon,
+        },
+    },
```

Comment:
The background of this state needs a little update on dark mode 
![image](https://user-images.githubusercontent.com/29347276/150242665-26ad119d-44d4-4748-9be3-f5250b2b9079.png)

Reply:
✅